### PR TITLE
update FAQ  description of how to use and develop EOS under Windows

### PR DESCRIPTION
**Pull request by ESEberhard**
(Work-in-Progress) This pull request is opened to start a discussion to provide a sufficient documentation for the usage of EOS on Windows.

### CODE REVIEW

File: doc/faq.rst [hunk 1]
```diff
@@ -34,11 +34,15 @@ Copy one of these links, making sure to include the lengthy token, and navigate
...
 ... use EOS on Windows?
 =======================
 
-Presently we do not support using EOS on Windows. We recommend that you install EOS on a
-remote Linux machine (e.g., one of your department's workstations) and connect remotely
-as discussed :ref:`above <... use EOS remotely?>`.
+EOS can be used on Windows by making use of the Windows Subsystem for Linux (WSL)
+which is offered and developed by Microsoft themselves.
+The installation of WSL is described in detail `here <https://docs.microsoft.com/en-us/windows/wsl/install>`__.
+This provides you with a well optimized virtual machine.
```
Comment (dvandyk) [inline doc/faq.rst:45]: Overly specific adjective: ``well optimized``. Instead: ``This provides you with a Linux container in virtual machine.``, maybe?

File: doc/faq.rst [hunk 2]
```diff
@@ -34,11 +34,15 @@ Copy one of these links, making sure to include the lengthy token, and navigate
...
+From here on, you can follow the instructions for the installation method of your choosing described for EOS under Linux.
+If you would like to make adaptions to the code for a project using EOS and/or contribute to its development,
+we recommend using VScode in WSL and following these `steps <https://code.visualstudio.com/docs/remote/wsl>`__.
```
Comment (dvandyk) [inline doc/faq.rst:48]:
```
[...] we recommend using Visual Studio Code as your development environment.
It supports connecting natively to the WSL container if you follow these `steps <...>`__.
```

**Review by dvandyk — COMMENTED** [review]
Looks good, I left some comments!

File: doc/faq.rst (hunk 1 above)
Comment (ESEberhard) [reply doc/faq.rst:45]: The point I intended to make with this phrasing, is that I feel like WSL is a far superior solution and comes with a way better user experience than a Virtual Machine using e.g. HyperV

**Comment by dvandyk** [comment]
@ESEberhard I cannot merge until you solve the conflic with @pluegh new FAQ entry.

**Comment by dvandyk** [comment]
Merged as part of #426.

**Pull request closed without merging**